Makefile

```diff
@@ -15,7 +15,7 @@ GOBUILD := GO111MODULE=on go build -v
 GOINSTALL := GO111MODULE=on go install -v
 GOMOD := GO111MODULE=on go mod
 
-COMMIT := $(shell git describe --abbrev=40 --dirty)
+COMMIT := $(shell git describe --abbrev=40 --dirty --tags --long)
```

Comment:
another option could be to add a separate `COMMIT_HASH` and then set that separately to the COMMIT one. 

but that of course would result in needing a new global variable & then a new `commit_hash` rpc field in GetInfo. 

So I guess we just need to decide if we want to keep the COMMIT working as it does today where if the "pure" tagged version is being run, then no commit hash is included in this variable. 

But if it doesnt matter, then this is probs the easiest solution. 

wdyt?

Reply:
Great idea, PTAL.